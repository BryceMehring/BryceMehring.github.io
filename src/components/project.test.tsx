import React from 'react';
import renderer from 'react-test-renderer';
import Project from './project';

describe('project', () => {
  it('matches snapshot', () => {
    const wrapper = renderer.create(
      <Project 
        name="name"
        shortDescription="short"
        position="pos"
        link="link"
        images={{
          thumbnail: 'thumbnail',
          alt: 'alt',
        }}
      />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  })
})