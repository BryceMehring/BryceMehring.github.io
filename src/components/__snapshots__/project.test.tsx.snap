// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`project matches snapshot 1`] = `
<div
  className="col-sm-12 col-md-4 col-xl-3 p-3 m-3 border shadow rounded project"
>
  <div
    className="text-center"
  >
    <h3>
      <a
        href="link"
      >
        name
      </a>
    </h3>
    <p>
      short
    </p>
    <p>
      <b>
        Position
      </b>
      : 
      pos
    </p>
  </div>
  <div
    className="thumbnail"
  >
    <a
      href="link"
    >
      <img
        alt="alt"
        className="img-fluid d-block m-auto"
        src="thumbnail"
      />
    </a>
  </div>
</div>
`;
